docs/_config.yml

```diff
@@ -47,11 +47,11 @@ categories:
   Frameworks:
     en: Frameworks
     cn: 计算框架
-    pt: Estruturas
+    pt: Frameworks
```

Comment:
Untranslating this doesn't save any letters, it's probably better to keep it in Portuguese